### ISSUE TITLE
Better looking close button

### ISSUE THREAD

**Issue by vertonghenb** [Blazored/Toast#29]
Currently the close button is pretty basic:
```html
<button>x</button>
```
an alternative could be:
```html
<a tabindex="0" class="blazored-toast-close">
    <i aria-label="icon: close" class="blazored-toast-close-icon">
        <svg viewBox="64 64 896 896" focusable="false" width="1em" height="1em" fill="currentColor" aria-hidden="true">
            <path
                d="M563.8 512l262.5-312.9c4.4-5.2.7-13.1-6.1-13.1h-79.8c-4.7 0-9.2 2.1-12.3 5.7L511.6 449.8 295.1 191.7c-3-3.6-7.5-5.7-12.3-5.7H203c-6.8 0-10.5 7.9-6.1 13.1L459.4 512 196.9 824.9A7.95 7.95 0 0 0 203 838h79.8c4.7 0 9.2-2.1 12.3-5.7l216.5-258.1 216.5 258.1c3 3.6 7.5 5.7 12.3 5.7h79.8c6.8 0 10.5-7.9 6.1-13.1L563.8 512z">
            </path>
        </svg>
    </i>
</a>
```
Notice the use of :
- classes
- aria labels
- tabindex
- svg 
  - [Optionally] Maybe make this a parameter?

I'm willing to PR this: (notes to self):
Todo:
- Adjust the close button
- Clean-up the css

Do note that this could be a breaking change if the client uses the following css selector:
`.blazored-toast-header button `

**Comment by chrissainty** [comment]
This is a great suggestion @vertonghenb and I'd be happy to take a PR for it. 

The only thing I'm not sure on is the need to swap from a `button` to an `a` tag. We could use the `i` inside the button just the same. It would also remove the potential breaking change you mentioned. Is there a specific reason it needs to be an `a`?

**Comment by vertonghenb** [comment]
No reason for it to be a <button> I'll file the pr later today.

**Comment by chrissainty** [comment]
Resolved by #31
